### PR TITLE
Added playbook for creation of custom role

### PR DESCRIPTION
**Pull request by sabre1041**
#### What is this PR About?
Documenting the creation of a custom role

#### Is there a relevant Trello card or Github issue open for this?
https://trello.com/c/7Waqfr66

#### Who would you like to review this?
cc: @redhat-cop/cant-contain-this

### CODE REVIEW

**Comment by itewk** [comment]
Looks awesome to me. Merge it :+1:

File: playbooks/operationalizing/custom_role_creation.adoc [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+:toc-title:
+
+OpenShift provides a fully functional link:https://docs.openshift.com/container-platform/3.4/architecture/additional_concepts/authorization.html[Role Based Access Control (RBAC)] system to govern the type of access users and groups have within the platform. While the platform does come with a series of roles that attempts to address the types of access applicable to the majority of users, there are cases in which a customization of the roles within OpenShift may be desired.     
+
+toc::[]
+
+
+== Overview
+
+The Role Based Access Controls within OpenShift gives platform administrators the tools to effectively govern the platform. Through the use of Roles, Policies and Bindings, operators can customize the level of access that are granted to each user of the platform. Access can be granted at a local (project) or cluster level. Several default roles are provided by the and include:
```
Comment (JaredBurck) [inline playbooks/operationalizing/custom_role_creation.adoc]: did not specify who provides the default roles.

File: playbooks/operationalizing/custom_role_creation.adoc [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+Since OpenShift already provides a set of default roles that fulfill the majority of the desired permission schemes, these can be used as a basis for any customized roles.
+
+NOTE: It is recommended that the default roles remain unaltered and instead new roles providing customized permissions be created 
+
+=== Determine the Resource
+
+The first step in restricting access to a particular function, the resource that provides the action must be determined. One of the methods in which to determine the resource is to increase the verbocity of the OpenShift Command Line (CLI) tool. Since every action (CLI or web console) makes use of the API, being able to trace the invocation affords the best opportunity to lcate the resource.
```
Comment (JaredBurck) [inline playbooks/operationalizing/custom_role_creation.adoc]: locate is misspelled

**Comment by sabre1041** [comment]
@JaredBurck applied updates per comments

**Comment by JaredBurck** [comment]
thanks for making these changes. LGTM

**Pull request merged**